clang/include/clang/Basic/BuiltinsRISCVCOREV.def

```diff
@@ -84,26 +84,26 @@ TARGET_BUILTIN(simd_dotup_h, "UZiUZiUZi", "nc", "xcvsimd")
 TARGET_BUILTIN(simd_dotup_b, "UZiUZiUZi", "nc", "xcvsimd")
 TARGET_BUILTIN(simd_dotup_sc_h, "UZiUZiUs", "nc", "xcvsimd")
 TARGET_BUILTIN(simd_dotup_sc_b, "UZiUZiUc", "nc", "xcvsimd")
-TARGET_BUILTIN(simd_dotusp_h, "ZiUZiSZi", "nc", "xcvsimd")
-TARGET_BUILTIN(simd_dotusp_b, "ZiUZiSZi", "nc", "xcvsimd")
+TARGET_BUILTIN(simd_dotusp_h, "ZiUZiUZi", "nc", "xcvsimd")
+TARGET_BUILTIN(simd_dotusp_b, "ZiUZiUZi", "nc", "xcvsimd")
 TARGET_BUILTIN(simd_dotusp_sc_h, "ZiUZiSs", "nc", "xcvsimd")
 TARGET_BUILTIN(simd_dotusp_sc_b, "ZiUZiSc", "nc", "xcvsimd")
-TARGET_BUILTIN(simd_dotsp_h, "ZiSZiSZi", "nc", "xcvsimd")
-TARGET_BUILTIN(simd_dotsp_b, "ZiSZiSZi", "nc", "xcvsimd")
-TARGET_BUILTIN(simd_dotsp_sc_h, "ZiSZiSs", "nc", "xcvsimd")
-TARGET_BUILTIN(simd_dotsp_sc_b, "ZiSZiSc", "nc", "xcvsimd")
+TARGET_BUILTIN(simd_dotsp_h, "ZiUZiUZi", "nc", "xcvsimd")
+TARGET_BUILTIN(simd_dotsp_b, "ZiUZiUZi", "nc", "xcvsimd")
+TARGET_BUILTIN(simd_dotsp_sc_h, "ZiUZiSs", "nc", "xcvsimd")
+TARGET_BUILTIN(simd_dotsp_sc_b, "ZiUZiSc", "nc", "xcvsimd")
```

Comment:
If my previous comment makes sense here all operands should be signed? Or am I reading wrong the operand specifiers for these simd instructions?

Reply:
Same as above.